### DISCUSSION TITLE
How can we improve?

### DISCUSSION THREAD

**Discussion by 93v** [93v/dynatron#14] [Ideas]
We are asking you to let us know how can we improve.

During your daily usage, you may have noticed some limitations or shortcomings that could be improved. We really encourage you to let us know any request or idea you have.

Thank you.

**Comment by karo-dc** [comment]
Hi, when doing update and requesting to get updated values via function "returnValues", the default value of function is "ALL_OLD", but in a lot of cases we are using "ALL_NEW" value, will be greate to chage default value to "ALL_NEW".

**Reply by gevorggalstyan** [reply]
Thank you @karo-dc this is a great idea!

**Reply by gevorggalstyan** [reply]
@karo-dc do you want to create a pull request changing this?

**Comment by troybESM** [comment]
For me it is more documentation on getting started. More examples on usage in a lambda would be nice as well.

**Reply by gevorggalstyan** [reply]
Thank you @troybESM we are working on a better, detailed and extended documentation. If you have any specific ideas to make it better please do not hesitate to create a pull request with your improvements.

**Comment by AbrahamChutilian** [comment]
Will be great, if the documentation is more begginer friendly and reach of IRL examples. For example in the usual usage part of the scan operation, the returned value will also be given.
